builder/docker/communicator.go

```diff
@@ -173,7 +173,7 @@ func (c *Communicator) UploadDir(dst string, src string, exclude []string) error
 
 	// Make the directory, then copy into it
 	cmd := &packer.RemoteCmd{
-		Command: fmt.Sprintf("set -e; mkdir -p %s; command cp -R %s/ %s",
+		Command: fmt.Sprintf("set -e; mkdir -p %s; command cp -R %s/. %s",
```

Comment:
aha, okay.  Got it.  I know eventually we want to rewrite this, and making it work for windows will be addressed then.